exampleCourse/questions/demo/autograder/r/createRObject/question.html

```diff
@@ -0,0 +1,25 @@
+<pl-question-panel>
+
+<markdown>
+
+----
+
+### Compound R Object
+
+Please create a single R object named `x` containing the following
```

Comment:
Nit for consistency with the other question:

```suggestion
Please create a single R object named `x` containing the following:
```

Reply:
I know you asked for it but I still don't like it.  I used this formulation and the SIGCSE workshop and I have a string inkling towards thinking the student prefer the object name closer to the end (as the "action item" to create). Anyway. Done.